src/DatePicker/index.jsx

```diff
@@ -7,70 +7,379 @@
  *
  */
 
-import React                        from 'react';
+import React, { useState }          from 'react';
 import PropTypes                    from 'prop-types';
+import moment                       from 'moment';
+import _                            from 'lodash';
 
+import copy                         from './copy.json';
 import DatePickerItem               from './DatePickerItem';
 import DatePickerHeader             from './DatePickerHeader';
 import { attachEvents, useTheme }   from '../utils';
 
 
+const DAY_LABELS = _.range( 0, 7 ).map( day => ( {
+    label : copy.dayHeaders[ day ],
+} ) );
+
+/**
+ * returns the timestamp of the current moment
+ *
+ * @return {Number} timestamp
+ */
+function now()
```

Comment:
Hmm not sure we need this function 🤔 Is there a difference between `new Date().getTime()` and `Date.now()`? If not, we could just use the latter directly

Reply:
Indeed, `Date.now()` is a bit [better](https://stackoverflow.com/questions/12517359/performance-date-now-vs-date-gettime?answertab=votes#tab-top) and there's no need to have `function now()`